linkerd/protocol/http/src/integration/scala/io/buoyant/linkerd/protocol/ForwardClientCertTest.scala

```diff
@@ -69,4 +70,49 @@ class ForwardClientCertTest extends FunSuite {
       }
     }
   }
+
+  test("clears incoming client certificate") {
+    var downstreamRequest: Request = null
+    val dog = Downstream.mk("dogs") { req =>
+      downstreamRequest = req
+      val rsp = Response()
+      rsp.contentString = "woof"
+      rsp
+    }
+
+    val linkerConfig =
+      s"""
+         |routers:
+         |- protocol: http
+         |  dtab: |
+         |    /p/dog => /$$/inet/127.1/${dog.port} ;
+         |    /svc/clifford => /p/dog ;
+         |  servers:
+         |  - port: 0
+         |  client:
+         |    kind: io.l5d.global
+         |""".stripMargin
+    val linker = Linker.load(linkerConfig)
+    val router = linker.routers.head.initialize()
+    val server = router.servers.head.serve()
+
+    val client = Upstream.mk(server)
+
+    try {
+      val rsp = {
+        val req = Request()
+        req.host = "clifford"
+        req.headerMap(ForwardClientCertFilter.Header) = "totally_bogus"
```

Comment:
It would be good to have a test where the value is syntactically valid to ensure that it is removed even when it looks good, and not only when it is "totally bugus."

Reply:
It would also be good to have a test that shows that if there are multiple instances of the header field that all of them are removed.